src/css/blocks/product-card.css

```diff
@@ -0,0 +1,122 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: var(--main-color);
+
+  box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, use `*` selector

Reply:
Done.
```
* {
  box-sizing: border-box;
}
```